slides/control_structures.tex

```diff
@@ -0,0 +1,182 @@
+				\caption*{(a) Increasing marginal costs}
+			\end{figure}
+		\end{column}
+		\begin{column}{0.5\textwidth}
+			\begin{figure}
+				\uncover<2->{
+					\input{../assets/figures/competitive_creation_const.tex}
+					\caption*{(b) Constant marginal costs}
+				}
+			\end{figure}
+		\end{column}
+	\end{columns}
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Monopolistic Creation}
+	\begin{figure}
+		\input{../assets/figures/monopolistic_creation.tex}
+		\caption*{Monopolistic money creation with constant marginal costs}
+	\end{figure}
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Physical Representation}
+	Physical monetary units are tied to an object.
+	\vspace{1em}
+	\begin{figure}
+		\centering
+		\includegraphics[height = 0.15\textheight]{../assets/images/currency}
+	\end{figure}
+	\vspace{1em}
+	\begin{columns}
+		\begin{column}{0.5\textwidth}
+			\uncover<2->{
+				\textbf{Pros:}
+				\begin{itemize}
+					\item Simple
+					\item Clear ownership rights
+					\item Anonymity
+					\item No systemic dependency
+				\end{itemize}
+			}
+		\end{column}
+		\begin{column}{0.5\textwidth}
+			\uncover<3->{
+				\textbf{Cons:}
+				\begin{itemize}
+					\item Location bound
+					\item Safekeeping and transport
+					\item Counterfeiting
+					\item Divisibility
+				\end{itemize}
+			}
+		\end{column}	
+	\end{columns}
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Virtual Representation}
+	Virtual monetary units allow the transfer of value without any change in the physical control of a particular object.
+	\vspace{1em}
+	\begin{figure}
+		\centering
+		\includegraphics[height = 0.15\textheight]{../assets/images/virtual}
+	\end{figure}
+	\vspace{1em}
+	\begin{columns}
+		\begin{column}{0.5\textwidth}
+			\uncover<2->{
+				\textbf{Pros:}
+				\begin{itemize}
+					\item No physical handover
+				\end{itemize}
+			}
+		\end{column}
+		\begin{column}{0.5\textwidth}
+			\uncover<3->{
+				\textbf{Cons:}
+				\begin{itemize}
+					\item Contestable
+				\end{itemize}
+			}
+		\end{column}	
+	\end{columns}
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Transaction Processing}
+	\begin{columns}[T]
+		\begin{column}{0.5\textwidth}
+			\center
+			\textbf{Decentralized}
+			\begin{figure}
+				\input{../assets/figures/dec_process.tex}
+			\end{figure}
+		\end{column}
+		\begin{column}{0.5\textwidth}
+			\uncover<2->{
+				\center
+				\textbf{Centralized}
+				\begin{figure}
+					\input{../assets/figures/cen_process.tex}
+				\end{figure}
+			}
+		\end{column}
+	\end{columns}
+	\vspace{1.5em}
+	\uncover<3->{
+	\vspace{1.5em}
+	For virtual monetary units:
+	\begin{center}
+			\begin{tabular}{lccc}
+				\hline \hline
+					& Decentralized & Centralized \\
+				Transaction capacity  & \uncover<6->{\checkmark} & \uncover<4->{\checkmark} \\
+				Transaction legitimacy & \uncover<6->{\checkmark} & \uncover<4->{\checkmark} \\
+				Transaction consensus & \uncover<6->{\textcolor{focus}?} & \uncover<4->{\checkmark} \\ \hline \hline
+			\end{tabular}
+		\end{center}
+	}
+\end{frame}
+%%%
+
```

Comment:
If we first mention Yap in the context of the cube's upper right corner and then revisit the cube with Bitcoin, there should be a slide that describes the differences, i.e. that Yap relies on reputational effects and only works in small communities, whereas Bitcoin is designed to work in a global and pseudonymous context. This is a version of the (German ppt) slide I usually work with to explain the relationship between the two:
![image](https://user-images.githubusercontent.com/8560663/109466540-28872680-7a6a-11eb-8208-d24177dbbd85.png)

Reply:
Thank you for the feedback, @Chainomics. I added a similar slide in commit c8de0eb.